### ISSUE TITLE
Viewability Rate & VTR problem.

### ISSUE THREAD

**Issue by TRixes** [googleads/googleads-ima-flash#1]
Hi,

We built a custom player with Videojs 4.12.x & Google IMA SDK. First time we used HTML SDK of IMA but viewability rate went to deep like 1-10 percent. It was impossible but then we noticed that fullplay and 100% percent trackings aborted by IMA. We decide, HTML5 SDK caused of this problem and we built with it Flash IMA SDK and we wrote a plugin for Videojs.

After this change our Viewability rate increased about 90 percent and VTR was fixed to 60 - 80 percent. But this week Viewability rate going down 60 - 80 percent and VTR going down under 40 percent. We controlled trackings and no problem about trackings.

We can't realize the problem. Do you have an idea about this?

**Comment by shawnbuso** [comment]
Are you seeing any specific issues on your site other than viewability numbers? Do you have any error logging in place that could clue us into what's going on? I can't say what would be causing this without seeing some kind of logs.

**Comment by TRixes** [comment]
You can see player at our website http://www.acunn.com/video. Player doesnt throw any error. Thanks for interest. And I can send for you some viewability reports.

**Comment by shawnbuso** [comment]
I'm not seeing ads in that player - are they locked to a specific country? Also, how does your viewability recording work? I noticed that when I load the page the player starts to load, but then I get sent to a fullscreen ad that I need to close to get back to the page. Could it be that users are just abandoning the page when they get to this fullscreen ad?

**Comment by TRixes** [comment]
Yes it is locked to country (Turkey). We use dfp premium at backend. About fullscreen ad, it is countdown ad but yes, some technical issues i couldnt write player pause function yet. May be this affected my situation. I'm looking for this. Thanks.
